# Week 10: Descriptive Inference (Solutions)

This week we will focus on using the `difference_in_means()` function to conduct descriptive inference. We will use the CCES survey data from after the 2020 election. 

As a reminder, this is what the variables mean: 

- `vvweight_post`: Survey weight
- `person_of_color`: `1` if person identifies as a person of color; `0` if person identifies as non-Hispanic white
- `college`: `1` if the person graduated from college; `0` if the person did not
- `female`: `1` if the person identifies as female; `0` otherwise
- `medicare_expand`: `1` if the person favors expanding Medicare; `0` otherwise
- `vote_wait`: How long someone said they had to wait to vote. Rounded to 0, 5, 15, 45, or 90 minutes. `NA` means missing (question was not asked), `difference_in_means()` will ignore these cases.
- `votereg_problem`: Encountered a problem when they tried to vote (e.g., ID was rejected, didn't appear on voter registration list); `0` = no; `1` = yes

In [1]:
#load the package and data 
library(estimatr)

data <- read.csv("ps3_cces2020_post.csv")
head(data)

vvweight_post person_of_color college female medicare_expand vote_wait
1 0.6065928     0               1       0      0               NA       
2 0.5533137     1               1       0      1               NA       
3 0.9029812     0               0       1      1               NA       
4 0.8359022     0               1       1      1               15       
5 0.3900869     0               0       1      1               NA       
6 0.8777516     1               0       1      1                0       
  votereg_problem
1 0              
2 0              
3 0              
4 0              
5 0              
6 0

What is the relationship between being a person of color and having trouble voting? 

Calculate the weighted difference in means using the `person_of_color` and `votereg_problem` variables. As a reminder, when the group variable is binary (that is, 1/0) you do not need to specify condition1 and condition2. The function will calculate the likelihood of encountering trouble voting if you are a person of color versus if you are not a person of color. 

In [5]:
#remember to add an argument for the survey weights
poc.vote.trouble <- difference_in_means(votereg_problem ~ person_of_color, data, weight = vvweight_post)
poc.vote.trouble

Design:  Standard (weighted) 
                  Estimate Std. Error  t value  Pr(>|t|)     CI Lower
person_of_color 0.01100867 0.00685321 1.606353 0.1082332 -0.002425268
                  CI Upper   DF
person_of_color 0.02444261 8580

## Interpretation

Reminder for what each number in the output 
- The **estimate** gives us the difference between the groups in this sample, and is our best guess about what the differences between the groups are in the population.
- **Standard errors** measure *how noisy* our estimate is. In this case noise is generated by the fact that our random sample might have randomly sampled a very slightly unrepresentative group.
- **$p$-values** measure the probability we would see a difference between the two groups as large as we did even if, in the population, the groups were exactly the same.
- In 95% of random samples, the **confidence intervals** will contain the true difference between the groups in the population.

Based on this, interpret your: 
- Estimate: Relative to those who don't identify as POC, being a POC is associated with a 1.1 percentage point higher likelihood of encountering trouble while voting.
- P-value: There is a 10.8 percentage chance of observing an estimate as big or bigger than 0.011 if there was no difference in encountering trouble while voting being a POC and not being a POC.
- 95% confidence interval: Over repeated samples, there is a 95% chance that the true difference between the groups lies between -0.002 and 0.02.